mne_bids/write.py

```diff
@@ -895,23 +901,42 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
                               'overwrite to True.' % bids_fname)
     _mkdir_p(os.path.dirname(bids_fname))
 
-    if verbose:
-        print('Copying data files to %s' % bids_fname)
-
-    if ext not in ALLOWED_EXTENSIONS:
-        raise ValueError('ext must be in %s, got %s'
-                         % (''.join(ALLOWED_EXTENSIONS), ext))
-
     # Copy the imaging data files
-    if ext in ['.fif']:
+    if ext not in ALLOWED_EXTENSIONS[kind]:
+        if kind == 'meg':
+            if verbose:
+                print('Converting data files to Functional Image Format (FIF)')
+            n_rawfiles = len(raw.filenames)
+            if n_rawfiles > 1:
+                split_naming = 'bids'
+                raw.save(bids_fname, split_naming=split_naming, overwrite=True)
+            else:
+                # This ensures that single FIF files do not have the part param
+                raw.save(bids_fname, overwrite=True, split_naming='neuromag')
+        else:
+            if verbose:
+                print('Converting data files to BrainVision format')
+            try:
+                from pybv import write_brainvision
```

Comment:
if pybv is used, we should add a note to the README, which currently says:
```
For full functionality of mne-bids, you will also need to pip install the following packages:

    nibabel

```

then the list of "nibabel" can be extended to include pybv

also, pybv needs to be added to the [`environment.yml`](https://github.com/mne-tools/mne-bids/blob/master/environment.yml) file